corehq/apps/smsbillables/models.py

```diff
@@ -353,3 +373,41 @@ def create(cls, message_log, api_response=None):
         billable.save()
 
         return billable
+
+    @classmethod
+    def _get_twilio_charges(cls, billable, message_log, backend_instance):
+        def _get_twilio_client(twilio_backend_id):
+            twilio_backend = SQLMobileBackend.get(twilio_backend_id)
+            account_sid = twilio_backend.account_sid
+            auth_token = twilio_backend.auth_token
```

Comment:
these properties should be accessed by setting `config = twilio_backend.config`, and then you can access `config.account_sid` and `config.auth_token`